flake.nix

```diff
@@ -32,14 +35,25 @@
         "aarch64-darwin"
       ];
 
+      rustToolchain = fenix.packages.x86_64-linux.fromToolchainFile {
+        file = ./rust-toolchain.toml;
+        sha256 = "sha256-X/4ZBHO3iW0fOenQ3foEvscgAPJYl2abspaBThDOukI=";
+      };
+
+      nixpkgsFor =
```

Comment:
note: currently this abstraction is not needed, however it's good to have a single source of truth for the `pkgs` used by all of the outputs and there is a decent chance we will need to this in the future.

(I added it to more add global overlays to `pkgs`, however I didn't end up doing that)